java/openjfx11/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem      1.0
 
 name            openjfx11
 version         11.0.2
-revision        0
+revision        1
```

Comment:
The `revision` should remain at 0. No one who has this port installed should have to re-install it because nothing has actually changed.

Reply:
Sure, makes sense.